mflowgen/core/run.py

```diff
@@ -163,10 +164,11 @@ def launch_run( s, design, update, backend ):
     c_dirname  = os.path.dirname( construct_path )
     c_basename = os.path.splitext( os.path.basename( construct_path ) )[0]
 
-    sys.path.append( c_dirname )
+    mod_spec = importlib.util.spec_from_file_location( c_basename, construct_path )
```

Comment:
Is this a better way than what was there before?

Reply:
Yes, this way is better because it explicitly points to the graph that we want to import, rather than relying on some path variables. I had issues with the wrong path being selected using the previous method.

Reply:
Oh that makes sense. The old way could have prioritized the same c_basename at some other PATH entry.